apps/explorer/src/hooks/useSearch.ts

```diff
@@ -65,37 +91,49 @@ const getResultsForAddress = async (rpc: JsonRpcProvider, query: string) => {
     if (!isValidSuiAddress(normalized) || isGenesisLibAddress(normalized))
         return null;
 
-    const [from, to] = await Promise.all([
-        rpc.getTransactions({ FromAddress: normalized }, null, 1),
-        rpc.getTransactions({ ToAddress: normalized }, null, 1),
-    ]);
-    if (from.data?.length || to.data?.length) {
-        return {
-            label: 'address',
-            results: [
-                {
-                    id: normalized,
-                    label: normalized,
-                    type: 'address',
-                },
-            ],
-        };
-    }
+    try {
+        const [from, to] = await Promise.all([
+            rpc.getTransactions({ FromAddress: normalized }, null, 1),
+            rpc.getTransactions({ ToAddress: normalized }, null, 1),
+        ]);
+        console.log(from.data.length, to.data.length);
+        if (from.data?.length || to.data?.length) {
+            return {
+                label: 'address',
+                results: [
+                    {
+                        id: normalized,
+                        label: normalized,
+                        type: 'address',
+                    },
+                ],
+            };
+        }
+    } catch (e) {}
+
     return null;
 };
 
 export function useSearch(query: string) {
     const rpc = useRpc();
+    const checkpointsEnabled = useFeature(
+        GROWTHBOOK_FEATURES.EPOCHS_CHECKPOINTS
+    ).on;
 
     return useQuery(
         ['search', query],
         async () => {
             const results = await Promise.all([
```

Comment:
You know I just realized you could probably use `allSettled` here and then avoid the intermediate try/catch in every function, you'd just need to filter the results here.